src/features/user/forms/PasswordForm.js

```diff
@@ -14,18 +14,28 @@ export const PasswordForm = ({ Component, title, onDone }) => {
     // 2. Listen for those PasSwords from those inputs
     const oldPassword = event.target.elements.currentPasswordInput.value
     const newPassword = event.target.elements.newPasswordInput.value
-
+    const newSameSword = event.target.elements.newSameSwordInput.value
```

Comment:
**praise:** Good variable name with "same", 
**suggestion:** but I do need this to be newSamePassword (otherwise others might thing this is an error when possible bug fixing something else later)

Reply:
@olavea
Fix: newSamePassword #276
1bda7c8

Reply:
🎉